packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1154,10 +1198,13 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
     );
     scrollbarPainter = ScrollbarPainter(
       color: widget.thumbColor ?? const Color(0x66BCBCBC),
+      minLength: widget.minThumbLength,
+      minOverscrollLength: widget.minOverscrollLength,
```

Comment:
You could handle the null case here and in `updateScrollbarPainter` like this
```suggestion
      minOverscrollLength: widget.minOverscrollLength ?? widget.minThumbLength,
```

Reply:
I'm not sure why we need this change -- the null case is already being handled in the constructor of RawScrollbar.

Reply:
The null case is already being handled in the constructor of RawScrollbar. Why do we need a null check here?

Reply:
@Piinks Hi. Any updates on this? I know you're busy with other more important stuff, but this has been lying around for 20 days and it's mostly finished.

Reply:
I've made the changes.